components/definitions/global.d.ts

```diff
@@ -42,6 +42,9 @@ declare global {
     brave_adblock: {
       initialize: () => void
     }
+    playlist: {
```

Comment:
This is legacy style of communicating from c++ to JS. We should fire event instead.